automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -165,7 +165,8 @@ class RuntimeCreationDiskSpec
     val runtimeName = randomeName.copy(asString = randomeName.asString + "pd-spec") // just to make sure the test runtime name is unique
     val runtimeWithDataName = randomeName.copy(asString = randomeName.asString + "pd-spec-data-persist")
     val diskName = genDiskName.sample.get
-    val diskSize = genDiskSize.sample.get
+    val diskSize = DiskSize(110)
```

Comment:
Why can't we continue `genDiskSize.sample.get`ing where we could set `newDiskSize` with some delta?

Reply:
the exact disk size from `df -H` will always a bit different from what we set. So having the starting VM size vary will make the validation impossible